src-gui/src/main/resources/ui/index.html.tmpl

```diff
@@ -1,13 +1,13 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>OPEN KILDA</title>
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>OPEN KILDA</title>
   <base href="/openkilda/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="ui/favicon.ico">
-</head>
-<body>
-  <app-root></app-root>
-<script type="text/javascript" src="ui/runtime.js"></script><script type="text/javascript" src="ui/polyfills.js"></script><script type="text/javascript" src="ui/styles.js"></script><script type="text/javascript" src="ui/scripts.js"></script><script type="text/javascript" src="ui/vendor.js"></script><script type="text/javascript" src="ui/main.js"></script></body>
-</html>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="ui/favicon.ico">
+</head>
+<body>
+  <app-root></app-root>
+<script type="text/javascript" src="ui/runtime.js"></script><script type="text/javascript" src="ui/polyfills.js"></script><script type="text/javascript" src="ui/styles.js"></script><script type="text/javascript" src="ui/scripts.js"></script><script type="text/javascript" src="ui/vendor.js"></script><script type="text/javascript" src="ui/main.js"></script></body>
```

Comment:
No need to create template for this because when you build UI it will automatically create/update index.html

Reply:
Fixed.